modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -14,48 +14,53 @@ endif::[]
 |500 - Bad response from authorization server, or WSDL SOAP Fault error
 |===
 
-The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request.
+The Client ID Enforcement Policy restricts access to a protected resource by only allowing requests from registered client applications. The policy ensures that the client credentials sent on each request have been approved to consume the API.
 
-The policy validates the Client ID and, optionally, the Client Secret, of a client application created within an Anypoint Platform Organization.
+When a client application is registered in Anypoint Platform, a set of credentials consisting of a client id and client secret pair are generated. Once the client application requests access to an API, a contract is created between the application and that API. An API that is protected with a Client ID Enforcement policy will only be accessible to applications that have an approved contract.
 
-IMPORTANT: This policy will not validate successfully any client application credentials generated outside of the Anypoint Platform context.
+When you apply a Client ID Enforcement policy, access tracking of your API is enabled by reporting the client ID along with the analytics events. You can configure Mule runtime engine (Mule) to encrypt all sensitive information thar pertains to policies, contracts, and initialization data.
```

Comment:
typo in thar. There is no link on "See more" for encryption.

Reply:
Fixed typo! Texted you about the link!